src/platform/plugins/shared/data_view_editor/public/components/data_view_editor_flyout_content.tsx

```diff
@@ -210,7 +213,22 @@ const IndexPatternEditorFlyoutContentComponent = ({
   );
 
   if (isLoadingSources || !existingDataViewNames) {
-    return <EuiLoadingSpinner size="xl" />;
+    return (
```

Comment:
There is also another loading visible when js is being loaded:

https://github.com/elastic/kibana/blob/bb877cff7ef24c3f7295bdcdbdebb960c7db81b3/src/platform/plugins/shared/data_view_editor/public/components/data_view_editor_lazy.tsx#L17-L21

![May-12-2025 10-48-22](https://github.com/user-attachments/assets/0b8437ee-57a9-4966-83fe-f65c615a330e)

Reply:
This one was messing with me! Sometimes I saw it, sometimes I did not. Thanks for pointing out where it resides... these are tricky to find :)